README.md

```diff
@@ -121,14 +121,15 @@ NPM script.
 
 ### check-node-version
 
-Add a `check-node-version` task to package.json, providing the desired version of Node.js
-and NPM that you wish to enforce. Execute it as part of the `test` NPM script as well.
+Add a `check-node-version` task to package.json, providing the desired version of
+Node.js and NPM that you wish to enforce. Execute it as part of the `standards`
+NPM script as well.
 
 ```json
 {
    "scripts": {
       "check-node-version": "check-node-version --node 16.15.0 --npm 8.5.5",
```

Comment:
Do we still enforce Node versions this way? I vaguely thought we left "you're using the right version of node" to `nvm` now. Could be mistaken though 🤷‍♂️

Reply:
Ah, I see. For a time we were locking the Node.js version in private repos, but not in public repos that support multiple versions. I didn't realize that wasn't desired.